### PR TITLE
Hotfix/leaflet

### PR DESCRIPTION
**Pull request by christian-rli**
This is my fix for the "loading leaflet twice problem". It's how i fixed it on the server as well. Did you have a different / better solution @Bachibouzouk ?

### CODE REVIEW

**Comment by nesnoj** [comment]
Is this required when PR https://github.com/rl-institut/WAM/pull/75 will have been merged?

**Comment by henhuy** [comment]
My suggestion is, to simply put

`INSTALLED_APPS = ['leaflet']`

into fred/settings.py. This should do the trick together with my PR

File: app_settings.py [hunk 1]
```diff
@@ -7,8 +7,9 @@
 from wam import settings  # pylint: disable=import-error
 
 # add the package leaflet to the wam core project installed apps
-if os.name == 'posix':
-    settings.INSTALLED_APPS.append('leaflet')
+#if os.name == 'posix':
+#    if 'leaflet' not in settings.INSTALLED_APPS:
+#        settings.INSTALLED_APPS.append('leaflet')
```
Comment (Bachibouzouk) [inline app_settings.py]: You can delete those lines instead of commenting them :)

**Review by Bachibouzouk — APPROVED** [review]

File: app_settings.py (hunk 1 above)
Comment (Bachibouzouk) [reply app_settings.py]: The comment is what causes the errors in Travis, so this will fix the errors as well ;)

**Pull request merged**